machine_learning_hep/utilities_validation.py

```diff
@@ -30,6 +30,44 @@ def fillvalidationvsmult(dfevt, dfevtevtsel, df_reco):
     Create a TH3F histogram and fill it with three variables from a dataframe.
     """
     _ = len(df_reco)
+    hlist = []
+    def makenfill(nx, xl, xh, ny, yl, yh, X, Y):
```

Comment:
I would suggest to have variable names with more meaningful names. The package became so large that it otherwise might be hard to follow what is going on

Reply:
I plan to use `makefill2dhist` as defined in `utilities_plot.py`

Reply:
I think the comment applies more to the variables `nx, xl, xh, ny, yl, yh, X, Y`. They are indeed quite cryptic and I think they would not pass the pylint check.

Reply:
Yes, I fully agree, my comment related to the fact that it had to be reimplemented.